tests/ssl-opt.sh

```diff
@@ -32,9 +32,31 @@
 
 set -u
 
-# Limit the size of each log to 10 GiB, in case of failures with this script
-# where it may output seemingly unlimited length error logs.
-ulimit -f 20971520
+
+if [ -z "${BASH_VERSION:-}" ]
```

Comment:
Please don't require bash to run `ssl-opt.sh`.

Use `.` instead of `source`. `source` is a bash-specific name, the portable command to include another file is `.`.

Reply:
Done . cleaned the patches